lib/distanceMatrixAPI.js

```diff
@@ -0,0 +1,15 @@
+const client = require('superagent');
+
+module.exports = (origin, destination) =>
```

Comment:
Does this work without the wrapping brackets on the arrow function? Even if it does, I think for codebase consistency and general best practice, I'd add them. So it would look like:
```
module.exports = (origin, destination) => {
  // existing body code
};
```

Reply:
Done.  I updated the eslintrc to take this into account.

An arrow function without brackets is an implicit return.  I agree that including the brackets is much more readable and it's good to know that it is best practice.  Thanks!